_includes/integrations/react-tech-ref.md

```diff
@@ -40,6 +41,50 @@ or with Yarn
 $ yarn add @tinymce/tinymce-react
 ```
 
+### Using TinyMCE React integration in a Bootstrap dialog
+
+To use {{site.productname}} React instances inside Bootstrap UI dialogs, add the following React effect to a component that renders with the editor. This code is required because Bootstrap blocks all `focusin` calls from elements outside the dialog.
+
+#### Bootstrap 5
+
+> **Note**: For Bootstrap 5, the React effect contains no JQuery and does not support Microsoft Internet Explorer 11.
+
+```jsx
+useEffect(() => {
+  const handler = (e) => {
+    if (
+      e.target.closest(
```

Comment:
I find this formatting really hard to read and is rather inconsistent with the rest of the docs examples. Is this standard react style?

Reply:
Codesandbox does that indenting automatically on save (presumably because the editor window is narrow) and I just copied and pasted. I can reformat it into one line.

Reply:
I have improved the formatting.